Formula/fox.rb

```diff
@@ -47,14 +47,13 @@ class Fox < Formula
   def install
     # Needed for libxft to find ftbuild2.h provided by freetype
     ENV.append "CPPFLAGS", "-I#{Formula["freetype"].opt_include}/freetype2"
-    system "./configure", "--disable-dependency-tracking",
+    system "./configure", *std_configure_args,
                           "--enable-release",
-                          "--prefix=#{prefix}",
                           "--with-x",
                           "--with-opengl"
     # Unset LDFLAGS, "-s" causes the linker to crash
     system "make", "install", "LDFLAGS="
```

Comment:
```suggestion
    system "make", "install", "LDFLAGS=#{ENV.ldflags}"
```
Might be nicer to do this. `ENV.ldflags` is probably blank for now, but it might not always be.